index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Exercise 6 Pseudo-Class Exercise</title>
+    <link rel="stylesheet" type="text/css" href="pseudo.css">
+  </head>
+  <body>
+    <ul class="top">
+      <li class="block">1</li>
+      <li class="block">2</li>
+      <li class="block">3</li>
+    </ul>
+
+    <span class="inline">X</span>
+    <span class="inline">Y</span>
+    <span class="inline">Z</span>
+
+    <article class="n-child">
```

Comment:
According to the readme step 4 "In the article element with the class n-child, add a 1 pixel border around every even numbered section element." Currently you have a border around section element 2 and 4, but you don;t have one around element 0 which is an even number.